src/client/common/configSettings.ts

```diff
@@ -95,6 +96,11 @@ export class JupyterSettings implements IWatchableJupyterSettings {
     // Hidden settings not surfaced in package.json
     public disableZMQSupport: boolean = false;
     public verboseLogging: boolean = false;
+    public variableTooltipFields: JSONObject = {
```

Comment:
I don't know that I would sink a ton of time into this. But could we examine if we could make this a specific type, and not JSONObject? JSONObject captures like...everything I think. I would think that we could get a specific interface for the format that we expect here.

Reply:
I believe it can just be an object.